object.go

```diff
@@ -106,8 +106,19 @@ func (e *Encoder) encObject(v POJO) error {
 		err    error
 		clsDef classInfo
 	)
-
+	pojo, isPojo := v.(POJO)
 	vv := reflect.ValueOf(v)
+
+	// get none pojo JavaClassName
+	var nonePojoJavaName string
+	if !isPojo {
+		s, ok := pojoRegistry.registry[vv.Type().String()]
```

Comment:
there will be a concurrent access issue, I add a func  `loadPOJORegistry` in https://github.com/apache/dubbo-go-hessian2/pull/244/files

Reply:
I 'll merge master after you merge this pr. And fix again.

Reply:
@LaurenceLiZhixin u can fix it now

Reply:
fixed